src/vehicle.cpp

```diff
@@ -4762,6 +4882,9 @@ void vehicle::refresh()
     alternator_load = 0;
     camera_epower = 0;
     has_atomic_lights = false;
+    plow_engine_drag = 0;
```

Comment:
Wouldn't that prevent them from being used independently, or being switched off then? That doesn't sound realistic, or am I missing something else?